docs/source/user-guide/sql/aggregate_functions.md

```diff
@@ -19,50 +19,293 @@
 
 # Aggregate Functions
 
-Aggregate functions operate on a set of values to compute a single result. Please refer to [PostgreSQL](https://www.postgresql.org/docs/current/functions-aggregate.html) for usage of standard SQL functions.
+Aggregate functions operate on a set of values to compute a single result.
 
 ## General
 
-- min
-- max
-- count
-- avg
-- sum
-- array_agg
+- [avg](#avg)
+- [count](#count)
+- [max](#max)
+- [mean](#mean)
+- [min](#min)
+- [sum](#sum)
+- [array_agg](#array_agg)
+
+### `avg`
+
+Returns the average of numeric values in the specified column.
+
+```
+avg(expression)
+```
+
+#### Arguments
+
+- **expression**: Column to operate on.
```

Comment:
The argument can be any expression, not just a column

so it is perfectly ok to do `avg(col1 + col2)` or `avg(col / 50)` or whatever

This applies to all of the aggregate functions